inst/include/dust/dust_cpu.hpp

```diff
@@ -395,11 +393,10 @@ class dust_cpu {
 
   std::vector<size_t> index_;
   std::vector<dust::particle<T>> particles_;
-  std::vector<dust::shared_ptr<T>> shared_;
```

Comment:
I can't work out what work this vector was doing before?

Reply:
Seemingly nothing! I believe this was never removed from the class from some long-passed refactor?